src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_NotificationsScreen.tsx

```diff
@@ -104,10 +110,22 @@ export const Profile_NotificationsScreen = () => {
                             .notifications.emailToggle.noEmailPlaceholder,
                     )
               }
+              disabled={data?.email === ''}
               value={isConfigEnabled(config?.modes, 'mail')}
               onValueChange={(enabled) => handleModeToggle('mail', enabled)}
             />
           </Section>
+          {data?.email === '' && (
```

Comment:
Is the thought here that if `data` is `undefined` then this button shouldn't be shown? In that case I think it is a little to "hidden", and kinda hard to be aware of on later refactors.

A tip when using `useQuery`, check for the status `success` (or use `isSuccess`), so you get proper typing on the returned data. 

Also I think we want to show the button if email is falsey, to not to be too brittle on later changes.

Reply:
Made some changes now. 
Email should always be set on the response, but if there is no email registered for the profile it will be empty string. `email: ''`
And that is the scenario (profile has no email) where we would like to show the button. 
With this in mind, was the change according to what you requested @gorandalum ? :)

Reply:
I think it is more clear :)